src/helpers/runWorkflow.js

```diff
@@ -0,0 +1,38 @@
+const sequence = require("when/sequence");
+const Table = require("cli-table2");
+const utils = require("../utils");
+const path = require("path");
+
+const logErrorToFile = (options, error) => {
+	const logPath = path.join(__dirname, "../../logs/.log.txt");
+	let content = "";
+	if (utils.fileExists(logPath)) {
+		content = utils.readFile(logPath);
+	}
+	content = `${content}\n
+${new Date().toLocaleString()}
+	version: ${options.version}
+	command: ${options.command}
+	step: ${options.step}
+	error: ${error.message}`;
+	utils.writeFile(logPath, content);
```

Comment:
This could just be...

```
const fs = require("fs");
fs.appendFileSync(logPath, content);
```

Then you could remove lines 9-12. At some point in the future maybe you could pull that out to a utils.appendFile method, but for now I'm find if you just use fs directly since the API is the same. It could be refactored in a following PR and update the other places you mentioned that could benefit for appendFile instead of writing the whole thing again